.github/workflows/publish-image.yaml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches:
+    - main
+
+name: Publish Latest Trickster Image
+
+jobs:
+  publish-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get Build Date
+        id: builddate
+        run: echo "::set-output name=builddate::$(date -u +%Y-%m-%dT%H:%M:%SZ)"
+      - uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: build & push image
+        run: |
+          docker buildx build \
+            --label "org.opencontainers.image.revision=${{ github.sha }}" \
+            --label "org.opencontainers.image.source=https://github.com/${{ github.repository }}" \
+            --label "org.opencontainers.image.created=${{ steps.builddate.outputs.builddate }}" \
+            --platform linux/arm64/v8,linux/amd64 \
+            --push \
+            -f ./Dockerfile \
+            -t ${{ env.PROXY_TAG_PREFIX }}:main \
+            -t ${{ env.PROXY_TAG_PREFIX }}:${{ github.sha }} \
+            -t ${{ env.IO_TAG_PREFIX }}:main \
+            -t ${{ env.IO_TAG_PREFIX }}:${{ github.sha }} \
+            .
+        env:
+          PROXY_TAG_PREFIX: tricksterproxy/trickster
```

Comment:
i recommend we only push to trickstercache/trickster (and not tricksterproxy/* or tricksterio/* going forward so that the 2.x release is only on the latest (and final) org.